### ISSUE TITLE
Bitwarden is not loading in Firefox

### ISSUE THREAD

**Issue by astridyu** [bitwarden/clients#2546]
### Steps To Reproduce

I have no idea how this can be reproduced.

### Expected Result

The regular Bitwarden window opens.

### Actual Result

The Bitwarden window does not open; instead, there is just a gray bar and you can't interact with it. You can't send passwords or anything.

### Screenshots or Videos

![Screenshot_20220504_103816](https://user-images.githubusercontent.com/7308591/166778025-42a8af9f-ec92-4df3-92a7-6f38db1c5e44.png)
![Screenshot_20220504_103846](https://user-images.githubusercontent.com/7308591/166778073-6afa5216-fad6-467e-b815-97b33a8c4bc1.png)

### Additional Context

No other extension is exhibiting this behavior (see above screenshot of React Developer Tools)

### Operating System

Linux

### Operating System Version

NixOS Unstable

### Web Browser

Firefox

### Browser Version

99.0

### Build Version

1.58.0 (gotten from Firefox extensions)

**Comment by Hinton** [comment]
Looks like the icon is missing as well?

Could you try reinstalling the extension? 

And if that doesn't work go to "Add-ons and themes", "Debug Add-ons" in the cog menu and click "Inspect" on the Bitwarden extension. You should then have access to inspect the background page and can view console errors.

To debug the popup extension you can select "Disable popup auto-hide"
![image](https://user-images.githubusercontent.com/137855/166905944-c8a237f5-7ab6-4e39-a3c7-7532a2104cac.png)

And open the popup to check if any errors occurs.

**Comment by astridyu** [comment]
I've reinstalled it several times, even did `rm -rf ~/.cache/mozilla ~/.mozilla` and tried it on another computer. It's only a problem with this computer.

I've also looked in the debug addons screen and there is nothing; no console info, no storage, no scripts. This is what inspector thinks it looks like, even with the sidebar.
![image](https://user-images.githubusercontent.com/7308591/167955991-ec9129bc-2107-4520-8078-60427fcbe3d8.png)
